examples/run_benchmarker.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""Entrypoint that runs the `Benchmarker` on a provided YAML hparams file."""
+import argparse
+import logging
+
+import composer
+from composer.benchmarker.benchmarker import Benchmarker
+from composer.benchmarker.benchmarker_hparams import BenchmarkerHparams
+from composer.trainer.trainer import Trainer
+from composer.trainer.trainer_hparams import TrainerHparams
+
+logger = logging.getLogger(__name__)
+
+
+def main() -> None:
```

Comment:
Note that this file is gonna need to get ran by some version of the distributed script. The `composer` script should work just fine if you want to use that :)